### ISSUE TITLE
Play sound after long-running command finished

### ISSUE THREAD

**Issue by nicolashohm** [warpdotdev/Warp#4155]
### Describe the solution you'd like?

I'm already in love with the feature to get a desktop notification after a long-running command finished. An option to also play a sound after the command is done would be awesome!

### Is your feature request related to a problem? Please describe.

I hate long-running commands, so I go grab a coffee. Unfortunately, I don't know when the command is done, so I can go back to my computer. 

### Additional context

I think a similar request was already reported as a bug #3727 

### How important is this feature to you?

2

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by nicolashohm** [comment]
Possible workaround I'm using right now: having the following alias `alias done="afplay /System/Library/Sounds/Blow.aiff"` and then running commands like this: `make build && done`

**Comment by hugows** [comment]
Yeah, I've been doing "Say done" but the feature is there for Warp to add a cool customizable sound (one for errors, one for success)

**Comment by Ben-Epstein** [comment]
I'm using this to get a sound when commands complete, which is great for me. But i'd love to be able to customize the sound. Right now it uses my mac's default notification sound, but being able to make it something unique to Warp (like slack) would be really helpful

<img width="401" height="198" alt="Image" src="https://github.com/user-attachments/assets/a241f39e-f72e-4abb-b32b-c87a09bda2fe" />
